fs/_ftp_parse.py

```diff
@@ -138,12 +145,22 @@ def decode_linux(line, match):
     return raw_info
 
 
+def _decode_windowsnt_time(date, time):
+    while len(time.split(":")[0]) < 2:
```

Comment:
Seems unlikely that `len(time.split(":")[0])` would ever be 0, so I guess the `while` could be changed to an `if` ?

Reply:
Done